all/pom.xml

```diff
@@ -29,6 +29,7 @@
   <name>Netty/All-in-One</name>
 
   <properties>
+    <javaModuleName>io.netty.all</javaModuleName>
```

Comment:
wouldn't it be better to not define a module at all if it not ships any classes ? 

/cc @shs96c

Reply:
As long as this jar doesn't export any classes, it'll be fine. The problems with the module system start when the same class is present in two or more jars: split packages (and therefore duplicate classes) are strictly forbidden.

Reply:
ok then it is fine.. Thanks!

Reply:
> wouldn't it be better to not define a module at all if it not ships any classes ?

Like I said above, I tried to make that happen, couldn't get it to work.